### DISCUSSION TITLE
NOT WORKING: row-count via postgREST API

### DISCUSSION THREAD

**Discussion by amypgon** [supabase/supabase#13195] [Questions] [answered]
(RLS is disabled)

SQL `SELECT COUNT(*) FROM "table1";` works just fine.

API-REQUEST
`curl --location --head 'https://X.supabase.co/rest/v1/table1?select=*' \
--header 'apikey: X' \
--header 'Count: Exact' \
--header 'Prefer: count=exact' \
--data-raw '{ count: exact }'`

API-RESPONSE
`1`

The response is incorrect. What I'm doing wrong?

**Comment by steve-chavez** [comment, marked as answer]
Only `Prefer: count=exact` is required. Your request seems wrong and `curl` will fail when doing a `--head` plus `--data-raw`. See the [docs](https://postgrest.org/en/stable/api.html#exact-count) for more details.
